styles/mixins/patterns/_molecules.scss

```diff
@@ -15,13 +16,18 @@ $-space: var.$layout-space;
 /**
  * Give an element a border, background color and internal vertical spacing
  */
-@mixin frame($with-hover: false) {
+@mixin frame {
   @include atoms.border;
   @include layout.vertical-rhythm;
 
   padding: $-space;
   border-radius: $-border-radius;
   background-color: $-color-background;
+
+  @include themes.theme('clean') {
+    // A frame should not have any borders in the clean theme.
```

Comment:
Both this comment and the one further down in this module are "x is 5"-type comments, but the point I'm trying to make is that we should _document_ theme deviations and their intent.